exercises/tom_clayton/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,194 @@
+/**
+ * Import statements
+ */
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <algorithm>
+#include <cctype>
+#include <string.h>
+#include <vector>
+#include <map>
+
+/** Returns a string filepath from user input
+ *  @return string filepath
+ */
+
+std::string getInputFn() {
+  std::cout << "Please enter the path of the file to be analysed" << std::endl;
+  std::string fin;
+  std::cin >> fin;
+  return fin;
+}
+
+/** Split up a string and return a vector of words with
+ *  len <=4, and converted to lower case
```

Comment:
In my previous comment I asked you to change this from len <4 to len <=4
I now see that it should read "...return a vector of works with len >4". Please can you update